.travis.yml

```diff
@@ -11,11 +11,10 @@ before_install:
   - source conda4travis.sh
 
 install:
-  - conda config --set remote_max_retries 5
-  - conda config --set channel_priority strict
-  - conda create -n atlite python pip pytest
-  - conda env update -n atlite --file=environment.yaml
+  - conda install mamba=0.1.2 -c conda-forge # fix the version for faster installation
```

Comment:
I don't like fixing the version number, we would have to update this continously and might install non-recent packages.
Is this a necessary limitation (performance-wise) or would a less strict statement like  `mamba>=0.1.2` yield a similar improvement? @FabianHofmann

Reply:
Actually you're right! The improvement is sufficiently small. I'll remove the fixing